src/include/duckdb/parser/expression/constant_expression.hpp

```diff
@@ -18,8 +18,10 @@ class ConstantExpression : public ParsedExpression {
 public:
 	static constexpr const ExpressionClass TYPE = ExpressionClass::CONSTANT;
 
+private:
+	ConstantExpression();
+
 public:
-	DUCKDB_API ConstantExpression();
```

Comment:
This is the odd one out having the DUCKDB_API attribute, I am not sure whether was useful / handy. I went straight and just removed the attribute when moving to private, but maybe here it make sense to be actually public?